test/unixfs-format.spec.js

```diff
@@ -15,128 +15,156 @@ const symlink = loadFixture('test/fixtures/symlink.txt.unixfs')
 const Buffer = require('safe-buffer').Buffer
```

Comment:
```suggestion
const { Buffer } = require('buffer')
```
can we remove 'safe-buffer' dep pls ?